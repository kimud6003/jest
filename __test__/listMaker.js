function listMaker(index= 3){
  let list =[];

  for(let i=1;i<=index;i++){
    list.push(i);
  }
  return list
}
export default listMaker;