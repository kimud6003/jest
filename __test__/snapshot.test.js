import { describe, test, expect } from "@jest/globals";
import listMaker from "./listMaker";

describe("snapshot Test", () => {
  test("listMaker", () => {
    // expect(listMaker()).toBe([1,2,3]);
    expect(listMaker()).toMatchSnapshot();
  });

  test("listMaker", () => {
    // expect(listMaker()).toBe([1,2,3]);
    expect(listMaker(4)).toMatchInlineSnapshot(`
      [
        1,
        2,
        3,
        4,
      ]
    `);
  });
});
