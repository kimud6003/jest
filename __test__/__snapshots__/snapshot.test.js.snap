// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Test listMaker 1`] = `
[
  1,
  2,
  3,
]
`;
